modules/ROOT/pages/edifact/edifact-edi-connector.adoc

```diff
@@ -19,7 +19,7 @@ release notes is listed in the See Also section of this document.
 To use the EDIFACT EDI connector in a production environment, you must
 have purchased a MuleSoft license for Anypoint B2B.
 
-NOTE: The EDIFACT EDI Connector supports files up to 15MB. The EDIFACT EDI connector's memory requirement is roughly 40 to 1. For example a 1MB file will require up to 40MB of memory to process. This is not an exact figure; the value may vary according to the complexity of the mapping instructions. It is important to consider this memory requirement in conjunction with your TPS needs for large EDIFACT files.
+NOTE: The EDIFACT EDI Connector supports files up to 15MB, and the memory requirement is roughly 40 to 1. For example, a 1MB file requires up to 40MB of memory to process, so it's important to consider this memory requirement in conjunction with your TPS needs for large EDIFACT files. This is not an exact figure as the value may vary according to the complexity of the mapping instructions.
```

Comment:
I am not sure if "as" translates well. I suggest using "because" instead.

Reply:
This is a good catch, Diane, but because the main point is that the figure is inexact, I suggest using a semicolon instead, to give the first part equal, if not primary, importance.

Reply:
Thanks Kim. And thanks Diane.